src/components/reportArticle/Index.jsx

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/jsx-no-bind */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Modal, Form, Row, Col, Alert, Spinner } from 'react-bootstrap';
+import { reportArticle } from '../../redux/actions/reports';
+
+const timeout = ms => new Promise(res => setTimeout(res, ms));
```

Comment:
@leksyib why are we setting time out?

Reply:
@9jaswag  We want to close the modal after some seconds if the reporting is successful so the user can see the success message. We don't want to close it immediately.

Reply:
remember to fix this